content/ADR-186-place-identifiers.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: adr
+adr: 186
+title: Place Identifiers
+date: 2023-02-06
+status: Draft
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - 2fd
+---
+
+## Abstract
+
+<!--
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the document section. **Someone should be able to read only the abstract to get the gist of what this document is about in its current state.** Abstracts should be always up to date with the current state of the document.
+-->
+
+This document describe how a Places is defined and when a new Place is created and replace and existing one.
+
+## Context
+
+<!--
+Discuss and go into detail about the subject in question. Make sure you cover:
+- Why is this decision important
+- The urgency of the decision
+- Datapoints and related background information
+- Vocabulary and key terms
+-->
+
+As the platform and the protocol is growing, a few initiatives like the possibility to save, rank and recommend scenes are becoming essential tools to potent the social interaction and discoverability in Decentraland. But, each new scene deploy generates a new ID in the catalyst which would reset every that the scene could collect on each deployment.
+
+The next approach was to track only *x,y* positions, but since a scene can grow, shrink, and change shape, this could result in misleading behavior for our users when, suddenly, a position they were already keeping track is not the scene they were following.
+
+Instead of the previously mentioned, we will assign a virtual UUID to each new deployment and use a set of rules detailed in the following section to determine if this new deployment corresponds to the same place or to a new one, in which case a new virtual ID will be assigned
```

Comment:
```suggestion
Instead of the previously mentioned approach, we will assign a virtual UUID to each new deployment and use a set of rules detailed in the following section to determine if this new deployment corresponds to the same place or to a new one, in which case a new virtual ID will be assigned.
```